judo-ui-react/src/main/resources/actor/src/pages/actions/action/call-operation-action/without-input-form.fragment.hbs

```diff
@@ -47,11 +47,13 @@ return async function {{ actionFunctionName action }} ({{# if action.operation.i
 
     {{ else }}
         {{# if (hasConfirmation action) }}
-            // TODO: implement shiny MUI Dialog here
-            const result = confirm(t('{{ getTranslationKeyForAction action }}.confirmation', { defaultValue: '{{ action.confirmationMessage }}' }) as string);
+            if (!silentMode) {
+                // TODO: implement shiny MUI Dialog here
```

Comment:
👀

Reply:
You know this is not a new addition right? :)